src/libraries/Microsoft.PowerFx.Core/FeatureFlags.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.PowerFx.Preview
     public static class FeatureFlags
     {
         private static bool _stringInterpolation = false;
+        private static bool _disableAtTableField = false;
```

Comment:
naming nit: This feature is more accuately called Row Scope Disambiguation Syntax, calling it AtTableField doesn't really mean much of anything.